webapp/components/suggestion/suggestion_box.jsx

```diff
@@ -182,6 +193,8 @@ export default class SuggestionBox extends React.Component {
 
         // Don't pass props used by SuggestionBox
         Reflect.deleteProperty(props, 'providers');
+        Reflect.deleteProperty(props, 'preventDefaultSubmit');
```

Comment:
Can you remove this line and https://github.com/mattermost/platform/blob/master/webapp/components/channel_switch_modal.jsx#L161 instead? `preventDefaultSubmit` isn't the name of a real prop so the warning was correct here